libs/common/src/lib/policies/site.policy.ts

```diff
@@ -0,0 +1,49 @@
+import { Site, User } from "@terramatch-microservices/database/entities";
+import { FrameworkKey } from "@terramatch-microservices/database/constants/framework";
+import { UserPermissionsPolicy } from "./user-permissions.policy";
+
+export class SitePolicy extends UserPermissionsPolicy {
+  async addRules() {
+    if (this.permissions.includes("sites-read") || this.permissions.includes("view-dashboard")) {
+      this.builder.can("read", Site);
+    }
+
+    const frameworks = this.permissions
+      .filter(name => name.startsWith("framework-"))
+      .map(name => name.substring("framework-".length) as FrameworkKey);
+    if (frameworks.length > 0) {
+      this.builder.can("read", Site, { frameworkKey: { $in: frameworks } });
+    }
+
+    if (this.permissions.includes("manage-own")) {
```

Comment:
For `manage-own`, we should also be checking projects in the org, per the current PHP policy. I did this recently for `project-report.policy.ts`, and you can probably just copy that code over.